_posts/2021-04-22-introducing-ugrcodes.md

```diff
@@ -0,0 +1,42 @@
+
+Inspired by a suggestion from one of our colleagues at the Wasatch Front Regional Council, we believe these codes provide several advantages that make them more useful and easier to use than FIPS codes or municipality names for uniquely identifying municipalities in Utah. They also support our efforts to ensure that attribute domain values match their corresponding descriptions within our datasets. Some advantages of the UGRCodes are:
+
+- Much more intuitive than FIPS codes    
+- Provide a unique identifier for municipalities
+- Avoid the need to account for differences in upper-, lower-, and title-casing across different datasets
+- Provide a static field for referencing a municipality without having to worry about alternate names or representations (Saint George vs St George vs St. George)
+- Simplify joins and queries by implementing a clearer, more memorable key field
+
+> *Disclaimer:* These are not "official" codes. They are codes that we created to (hopefully) make life just a little easier in Utah. They are not blessed by the National Institute of Standards and Technology, or the U.S. Census Bureau, or the International Organization for Standardization, or the Utah Geographic Information Council, or the Utah League of Cities and Towns, or the Utah Brewer's Guild, or any other credible or dubious organization. Just the UGRC...hence the name, UGRCode.
+
+In coming up with the 3-letter codes we tried to follow a few guiding principles, which are listed below. However, in order to deconflict the codes, add clarity, or maintain continuity, we had to stray from these principles occasionally.
+
+- Align codes with 3-letter "City Codes" that already exist in 911 dispatch datasets wherever possible
+  - E.g., Beaver (BVR), Millcreeak (MLC), Saint George (STG)
+- Preferentially use the first 3 letters of a municipality's name
+  - E.g., Hinckley (HIN), Layton (LAY), Mapleton (MAP)
+- Incorporate the first letters of multiple-word municipality names
+  - E.g., Salt Lake City (SLC), West Valley City (WVC), Bear River City (BRC)
+- Distinguish municipalities with similar names or beginnings as much as possible
+  - E.g., Kanab (KNB), Kanarraville (KNV), Kanosh (KNS)
+- Metro Townships have "MT" in their code - or just "T" if new ones conflict with current codes
+  - E.g., Emigration Canyon (EMT), Kearns (KMT), Magna (MMT)
+  
+#### Future Enhancements
+In the future, we may continue to expand on this idea by including the code field in additional datasets (road centerlines, address points, etc.). We believe UGRCodes could serve as a useful and intuitive code for querying and joining data across several datasets. The expanded use of this field may depend on how useful we find the codes, your feedback as you use the codes, or more. So reach out to us and let us know what you think via email ([agrc@utah.gov](mailto:agrc@utah.gov)) or Twitter ([@maputah](https://twitter.com/maputah)).

```

Comment:
**chore**: we don't allow email addresses directly in code. For reasons like the impending agrc@utah.gov to ugrc@utah.gov name change. So please switch this to use the contacts.yml file with help from the [readme](https://github.com/agrc/gis.utah.gov/blob/master/README.md#contact-information)

Reply:
Ahh, thanks.  I was wondering when we'd be changing the email address, too.

Reply:
This was a bit awkward because I couldn't find where the autopopulated text came from (hence the trial and error), but I think I eventually got it into something workable.  Although it still references "staff from AGRC".

Reply:
@steveoh please take another look at this PR when you get the chance